### PR TITLE
move quickstart to separate page and add kmesh l7 content

### PR DESCRIPTION
**Pull request by YaoZengzeng**
**What type of PR is this?**

**What this PR does / why we need it**:

1. Move quickstart to separate page to make README more concise
2. Add Kmesh L7 related content to quickstart

**Which issue(s) this PR fixes**:
Fixes part of #135 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/kmesh_quickstart-zh.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  - Kmesh当前是对接Istio控制面，启动Kmesh前，需要提前安装好Istio的控制面软件；具体安装步骤参考：https://istio.io/latest/docs/setup/getting-started/#install
+  - 完整的Kmesh能力依赖对OS的增强，需确认执行环境是否在Kmesh支持的[OS列表](./kmesh_support-zh.md)中，对于其他OS环境需要参考[Kmesh编译构建](../docs/kmesh_compile-zh.md)；也可以使用[兼容模式的kmesh镜像](../build/docker/README.md#兼容模式镜像)在其他OS环境中进行尝试，关于kmesh各种镜像的说明请参考[详细文档](../build/docker/README.md)。
+  
+- Docker镜像：
+
+  Kmesh实现通过内核增强将完整的流量治理能力下沉至OS。当发布镜像时，镜像适用的OS的范围是必须考虑的。因此，我们考虑发布三种类型的镜像：
+
+  - 支持内核增强的OS版本：
+
+    当前[openEuler 23.03](https://repo.openeuler.org/openEuler-23.03/)原生支持Kmesh所需的内核增强特性。Kmesh发布的镜像可以直接在该OS上安装运行。对于详细的支持内核增强的OS版本列表，请参见[链接](https://github.com/kmesh-net/kmesh/blob/main/docs/kmesh_support.md)。
```
Comment (LiZhenCheng9527) [inline docs/kmesh_quickstart-zh.md:14]:
```suggestion
    当前[openEuler 23.03](https://repo.openeuler.org/openEuler-23.03/)原生支持Kmesh所需的内核增强特性。Kmesh发布的镜像可以直接在该OS上安装运行。对于详细的支持内核增强的OS版本列表，请参见[链接](./kmesh_support-zh.md)。
```

File: README.md [hunk 2]
```diff
@@ -53,101 +53,7 @@ Note: * Planning
...
-  - Replace the waypoint image with the kmesh customized image.
-
-    ```
-    [root@ ~]# kubectl get gateways.gateway.networking.k8s.io
-    NAME      CLASS            ADDRESS         PROGRAMMED   AGE
-    default   istio-waypoint   10.96.143.232   True         5m7s
-    ```
-
-    Add annotation "sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint-{arch}:v0.3.0" to the `default` gateway, convert `{arch}` to the architecture of the host, current optional values are `x86` and `arm`. Then gateway pod will restart. Now kmesh is L7 enabled!
+Get started quickly, try building, deploying and experiencing the core features of Kmesh, please refer to [Quick Start](docs/kmesh_quickstart.md)
```
Comment (LiZhenCheng9527) [inline README.md:56]: Refer to 
```suggestion
Please refer to [Quick Start](docs/kmesh_quickstart.md)
```

File: README-zh.md [hunk 3]
```diff
@@ -53,102 +53,7 @@ Kmesh创新性的提出将流量治理下沉OS，在数据路径上无需经过
...
-  - 用kmesh自定义的镜像替换waypoint的原生镜像
-
-    ```
-    [root@ ~]# kubectl get gateway
-    NAME      CLASS            ADDRESS         PROGRAMMED   AGE
-    default   istio-waypoint   10.96.143.232   True         5m7s
-    ```
-
-    在`default` gateway的annotations当中添加`sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint-{arch}:v0.3.0`，将`{arch}`转换为所在宿主机的架构，当前可选的取值为`x86`和`arm`。在gateway pod重启之后，kmesh就具备L7能力了！
+快速入门，尝试构建、部署和体验Kmesh的核心功能，具体请参考[快速入门](./docs/kmesh_quickstart-zh.md)；
```
Comment (LiZhenCheng9527) [inline README-zh.md:56]:
```suggestion
请参考[快速入门](./docs/kmesh_quickstart-zh.md)文档；
```

**Comment by hzxuzhonghu** [comment]
This should be moved to kmesh-net/website now

File: README-zh.md (hunk 3 above)
Comment (nlgwcy) [reply README-zh.md:56]: Pls check whether needs to be updated after https://github.com/kmesh-net/kmesh/pull/327 merged.

**Comment by YaoZengzeng** [comment]
Already move to kmesh-net/website, closed

**Pull request closed without merging**